linux/keyman-config/Makefile

```diff
@@ -4,6 +4,7 @@ default: clean version man
 	python3 setup.py build
 
 install: # run as sudo
+	pip3 install qrcode
```

Comment:
This requires adding `python3-pip` to `Build-Depends` section in `debian/control` file.